packages/arb-util/configuration/configuration.go

```diff
@@ -194,6 +195,8 @@ func ParseValidator(ctx context.Context) (*Config, *Wallet, *ethutils.RPCEthClie
 func ParseNonRelay(ctx context.Context, f *flag.FlagSet) (*Config, *Wallet, *ethutils.RPCEthClient, *big.Int, error) {
 	f.String("bridge-utils-address", "", "bridgeutils contract address")
 
+	f.String("env-prefix", "", "environment variables with given prefix will be loaded as configuration values")
```

Comment:
Don't we want to also provide this for relays?

Reply:
We probably want this in `beginCommonParse`

Reply:
:+1: